### ISSUE TITLE
Improve examples formatting

### ISSUE THREAD

**Issue by Andersson007** [ansible-collections/community.rabbitmq#95]
##### SUMMARY
As was spotted in https://github.com/ansible-collections/community.rabbitmq/pull/73#discussion_r705131780, 
the format of the examples can be improved to
```
- name: Do something
  community.rabbitmq.rabbitmq_module:
    ...
```

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
All the modules need to be checked

**Comment by Andersson007** [comment]
@abompard would you like to pick it up as we spotted this in your PR?

**Comment by duttaANI** [comment]
I have found the following files in [modules](https://github.com/ansible-collections/community.rabbitmq/tree/main/plugins/modules) where in EXAMPLES  can be formatted as mentioned [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block)

1. rabbitmq_vhost.py
2. rabbitmq_global_parameter.py
3. rabbitmq_vhost_limits.py
4. rabbitmq_exchange.py
5. rabbitmq_binding.py
6. rabbitmq_queue.py

I would like to have permission to proceed with this task if possible.

**Comment by Andersson007** [comment]
@duttaANI hi and welcome to the project!
I think @abompard has not started with this yet, so feel free to proceed!
Thank you!

**Comment by abompard** [comment]
Yeah, thanks, I don't have much bandwidth for this at the moment unfortunately.

**Comment by duttaANI** [comment]
Should I also add **r** letter ? 
like EXAMPLES = **r** '''   .... ''' , after = symbol

**Comment by Andersson007** [comment]
> Should I also add **r** letter ?
> like EXAMPLES = **r** ''' .... ''' , after = symbol

yes, please add this, thanks!
